File: a-lunch-picker-program.js
const lunches = [];

function addLunchToEnd(array, item) {
  array.push(item);
  console.log(`${item} added to the end of the lunch menu.`);
  return array;
}

function addLunchToStart(array, item) {
  array.unshift(item);
  console.log(`${item} added to the start of the lunch menu.`);
  return array;
}

function removeLastLunch(array) {
  if (array.length === 0) {
    console.log("No lunches to remove.");
  } else {
    const removedItem = array.pop();
    console.log(`${removedItem} removed from the end of the lunch menu.`);
  }
  return array;
}

function removeFirstLunch(array) {
  if (array.length === 0) {
    console.log("No lunches to remove.");
  } else {
    const removedItem = array.shift();
    console.log(`${removedItem} removed from the start of the lunch menu.`);
  }
  return array;
}

function getRandomLunch(array) {
  if (array.length === 0) {
    console.log("No lunches available.");
  } else {
    const randomIndex = Math.floor(Math.random() * array.length);
    const selectedItem = array[randomIndex];
    console.log(`Randomly selected lunch: ${selectedItem}`);
  }
}

function showLunchMenu(array) {
  if (array.length === 0) {
    console.log("The menu is empty.");
  } else {
    console.log(`Menu items: ${array.join(", ")}`);
  }
}

File: a-lunch-picker-program.py
import math
import random

lunches = []


def add_lunch_to_end(array, item):
    array.append(item)
    print(f"{item} added to the end of the lunch menu.")
    return array


def add_lunch_to_start(array, item):
    array.insert(0, item)
    print(f"{item} added to the start of the lunch menu.")
    return array


def remove_last_lunch(array):
    if len(array) == 0:
        print("No lunches to remove.")
    else:
        removed_item = array.pop()
        print(f"{removed_item} removed from the end of the lunch menu.")
    return array


def remove_first_lunch(array):
    if len(array) == 0:
        print("No lunches to remove.")
    else:
        removed_item = array.pop(0)
        print(f"{removed_item} removed from the start of the lunch menu.")
    return array


def get_random_lunch(array):
    if len(array) == 0:
        print("No lunches available.")
    else:
        selected_item = random.choice(array)
        print(f"Randomly selected lunch: {selected_item}")


def show_lunch_menu(array):
    if len(array) == 0:
        print("The menu is empty.")
    else:
        print(f"Menu items: {', '.join(array)}")